site2/docs/sql-rest-api.md

```diff
@@ -0,0 +1,196 @@
+---
+id: sql-rest-api
+title: Pulsar SQL REST APIs
+sidebar_label: REST APIs
+---
+
+## Overview
+
+This describes the resources that make up the Presto REST API v1. If you have any problems or request, please create a Github issue to contact us.
+
+## Current version
+
+All requests should use the v1 version of REST API to request the presto service. You need to use explicitly URL `http://presto.service:8081/v1` to request service.
+
+Also, the header `X-Presto-User` is required when you send a `POST` request.
+
+```properties
+X-Presto-User: username
+```
+
+For more information about headers, see the [presto/PrestoHeaders.java at master · prestodb/presto · GitHub](https://github.com/prestodb/presto/blob/master/presto-client/src/main/java/com/facebook/presto/client/PrestoHeaders.java)
```

Comment:
"presto/PrestoHeaders.java at master · prestodb/presto · GitHub" 

why do we need "at master · prestodb/presto · GitHub"?